KlockanAPI/src/KlockanAPI.Presentation/Controllers/ProgramsController.cs

```diff
@@ -58,4 +58,25 @@ public async Task<ActionResult<ProgramDTO>> DeleteProgram(int id)
 
         return Ok(course);
     }
+
+    [HttpPut]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    public async Task<ActionResult<ProgramDTO>> EditProgram([FromBody] ProgramDTO editProgramDTO)
```

Comment:
What is the path for Edit Program? `api/v1/programs` ? 
It would be good practice for the Endpoint to be  `api/v1/programs` 
Send the id through the path and not the body